lib/viewcomponents/graph.jsx

```diff
@@ -1,68 +1,73 @@
 var React = require('react/addons');
 var classSet = React.addons.classSet;
 var mouseDownDrag = require('../utilities/mouseDownDrag');
+
+// Note: This might have the nasty side effect of overwriting the built-in
+// Node object. We might want to considering namespacing and/or renaming
+// this class.
```

Comment:
If @salva-abate merges before you do, first off I pity your merge conflicts, second I will help you with those, and third this problem will go away.